doc/notebooks/graphs-ipids.ipynb

```diff
@@ -550,7 +550,7 @@
      "collapsed": false,
      "input": [
       "%matplotlib inline\n",
-      "ans.multiplot(lambda (p, q): (q[IP].src, (q.time, q[IP].id)), plot_xy=True)"
+      "ans.multiplot(lambda p_q: (p_q[1][IP].src, (p_q[1].time, p_q[1][IP].id)), plot_xy=True)"
```

Comment:
I've fixed this one in a better way, see https://github.com/secdev/scapy/pull/1286

Reply:
I don't think so. You way is more elegant but it does not work (I'v checked changes using jupyter).

Reply:
I'm using a backward compatibility function to convert old python 2 lambdas into newer functions (see the addition in compat.py). It makes the lambdas way cleaner

Reply:
IMHO too much complicated and not worth not big benefits. :(

Reply:
Also iin CONTRIBUTING.md is:
### Python 2 and 3 compatibility
[...]
- lambdas must be written using a single argument when using tuples: use `lambda x_y: x_y[0] + f(x_y[1])` instead of `lambda (x, y): x + f(y)`.

Reply:
I know I wrote those lines. You'll admit that `lambda x,y: ...` is much cleaner

Reply:
...but for now FBC is IMHO most important/mandatory. Please think only about FBC /customers only think/dream such way / :)

Reply:
What I'm saying is that it DOES handle full backward compatibility.

We have not published a release since python 3 support, so all people are still using the
`lambda (x,y): ...` stuff, which IS CONVERTED TO THE NEW VERSION USING THE FUNCTION AVAILABLE IN COMPAT.py

Reply:
Cant agree.
Currently this lambda can be used as e.g.
plist.multiplot(lambda x: (x[1][IP].src, (x[1].time, x[1][IP].id))
so can be a function with one parameter which DOES NOT HAVE TO BE a tuple
and all works ok/as expected..
After your changes it is impossible to use such lambda (you'v changed params -> *params)
so FBC _has gone_ :(.

Reply:
You way/change IMHO is valuable but maybe for 2.4.1 after deep investigation of ev. (not obvious for now) impact for another things?

Reply:
Thanks for reporting the bug !
I’ve slightly modified the compat function so that it supports your case ! Could you check ?